src/corner_kick/config/webpack.prod.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Specifies the webpack configuration for production.
+ * Extends the general webpack configuration.
+ *
+ * @see {@link ./webpack.config.js}
+ */
+
+const path = require('path');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+
+const webpackConfig = require('./webpack.base.config.js');
+
+module.exports = {
+    ...webpackConfig,
+    mode: 'production',
+    devtool: false,
+    // Move node-modules to a separate individual bundles
+    optimization: {
+        runtimeChunk: 'single',
+        splitChunks: {
+            chunks: 'all',
+            maxInitialRequests: Infinity,
+            minSize: 0,
+            cacheGroups: {
+                vendor: {
+                    test: /[\\/]node_modules[\\/]/,
+                    name(module) {
+                        // get the name. E.g. node_modules/packageName/not/this/part.js
+                        // or node_modules/packageName
+                        const packageName = module.context.match(
+                            /[\\/]node_modules[\\/](.*?)([\\/]|$)/,
+                        )[1];
+
+                        // npm package names are URL-safe, but some servers don't like @ symbols
+                        return `npm.${packageName.replace('@', '')}`;
+                    },
+                },
+            },
+        },
+    },
+    // We run the visualizer locally so we can keep our bundles a bit bigger
+    // than the default max size
```

Comment:
Why is this a good thing to do?

Reply:
Added more clarification. We could make the bundle smaller through various performance fix, but
would make the build file more complex. Because we run locally, we do not have to worry too much about bundle size.

```
// Webpack complains that our bundle is too large.
// We run the visualizer locally so we can keep our bundles bigger
// than the default max size.
```